packages/yourloops/test/integration/assert/medical-widget.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { screen, within } from '@testing-library/react'
+import { type MedicalRecord } from '../../../lib/medical-files/models/medical-record.model'
+import { loggedInUserId } from '../mock/auth0.hook.mock'
+import { mySecondTeamId } from '../mock/team.api.mock'
+import { monitoredPatientId } from '../mock/patient.api.mock'
+import MedicalFilesApi from '../../../lib/medical-files/medical-files.api'
+import userEvent from '@testing-library/user-event'
+
+const MEDICAL_RECORD_TO_CREATE_ID = 'fakeMedicalRecordId'
+const MEDICAL_RECORD_TO_CREATE_DATE = '01-01-2023'
+
+const checkMedicalRecordCancel = async (medicalFilesWidget: HTMLElement): Promise<void> => {
+  const createMedicalReportButton = within(medicalFilesWidget).getByRole('button', { name: 'New' })
+  await userEvent.click(createMedicalReportButton)
+  const createdMedicalRecordDialog = screen.getByRole('dialog')
+  expect(createdMedicalRecordDialog).toHaveTextContent('Create medical record1. Diagnosis​2. Progression proposal​3. Training subject​CancelSave')
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Why `created` for MedicalRecordDialog ? an extra d ?